src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -122,6 +41,15 @@ internal class SemanticTokensHelpers
                 [ClassificationTypeNames.StringLiteral] = LSP.SemanticTokenTypes.String,
             };
 
+        public static readonly ImmutableArray<string> RoslynCustomTokenTypes = typeof(ClassificationTypeNames).GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static)
```

Comment:
Just curious, why don't we add a AllTypes property to `ClassificationTypeNames` instead of using reflection?